package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-aggregate-translations",
+  "version": "0.0.0",
+  "description": "The terra-aggregate-translations pre-build tool assists with creating the translation, intl loader and translation loader files that are configured for the specified locales. This tool is offered as a CLI script and as a setup function.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-aggregate-translations.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-aggregate-translations",
+    "translations",
+    "react",
+    "react-intl",
+    "i18n",
+    "intl"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/cerner/terra-aggregate-translations#readme",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-aggregate-translations/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.4.3",
```

Comment:
This project shouldn't need babel since the aggregate translations script will be run by node.

Reply:
Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/1730c5af56deebfe97427c6e78bea6c0ea32fb47